textworld/envs/wrappers/glulx_logger.py

```diff
@@ -35,63 +37,69 @@ def __len__(self) -> int:
     def current_game(self) -> list:
         """
         Gets current game we're logging.
+
         Returns: list of logs from current game.
         """
-        return self._current_game
+        return self._logs[-1]
 
     @property
     def logs(self) -> list:
         """
         Get all logs from all games.
+
         Returns: All logs from all games.
         """
         return self._logs
 
-    def new_game(self):
+    def new_game(self) -> list:
         """
         Start logs for a new game.
+
         Returns: log object for current game.
         """
-        if len(self._current_game) > 0:
+        if len(self.current_game) > 0:
```

Comment:
What do you need this check?

Reply:
We need this check to see if the current game is already new. We initialize our logs with a game inside, so there would be a redundancy if we didn't do this check.